CruzHacks/cruzhacks-2024-website
# Feature/homepage

# Scope

- Missing informational homepage as [mocked up in Figma](https://www.figma.com/file/IXAajiFoWwcPU70DFs6hRt/2024-Website?type=design&node-id=694-2422&mode=dev)

# Implementation

- Created Sections
- Added ScrollToAnchor functionality and Navbar
- Added embellishments (clouds / stars)
- Fade in Landing Transition

# How to Test

- Navigate to home page
- Test Navbar Links
- Check logged in and logged out Navbar/landing state

# TODO:

- [x] Mailchimp Subscribe backend endpoint
- [ ] Get Sponsors from Sponsorship
- [ ] Get Speakers from Logistics?/Sponsorship?
- [ ] Get Volunteer Application form
- [ ] Update 2023 -> 2024 linked google docs

## Nice to Have TODO:

- [x] Fix scrollTo Anchor points
- [ ] Implement Threejs speaker scroller
- [x] Change Discord wrap on HeroBlurb
